### ISSUE TITLE
String typehint will cause php 5.6 exception

### ISSUE THREAD

**Issue by xiaohuilam** [tymondesigns/jwt-auth#1774]
## Environment
- Laravel 5.3
- PHP 5.6.3
- [tymon/jwt-auth 1.0.0-rc.4](https://github.com/tymondesigns/jwt-auth/releases/tag/1.0.0-rc.4)

## Error
```bash
Error Argument 1 passed to Tymon\JWTAuth\Providers\Storage\Illuminate::setLaravelVersion() must be an instance of Tymon\JWTAuth\Providers\Storage\string, string given, called in ...
```

## Code
https://github.com/tymondesigns/jwt-auth/blob/d0d67c052bc13db84ce910aa8b89145b342c8e6d/src/Providers/Storage/Illuminate.php#L147-L148

--

and as another issue https://github.com/tymondesigns/jwt-auth/issues/1773 referenced, travis phpunixt does not covered these code because travis.ci says build [passed](https://travis-ci.org/tymondesigns/jwt-auth/jobs/505655321).

**Comment by tymondesigns** [comment]
ah my mistake! Been so long since I've worked with php 5. Will push up a fix

**Comment by tymondesigns** [comment]
Closed by 1ce697e1fa280bf22196bc397b324bc9997b90b3
